Project-I-Social-Card/social-card/src/App.js

```diff
@@ -1,22 +1,15 @@
-import React from 'react';
-import './App.css';
+import React from "react";
+import "./App.css";
+import HeaderContainer from "./components/HeaderComponents/HeaderContainer";
+import CardContainer from "./components/CardComponents/CardContainer";
 
 const App = () => {
   return (
     <div>
-      <h3>Welcome to React Social Card!</h3>
-      <p>
-        Begin by exploring the `components` directory. You'll notice we have a
-        few files that we've already included in there to get you started right
-        away building components. You'll need to make sure you include your
-        components that you build in this file to watch your app come to life
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+      <div className="social-card">
+        <HeaderContainer />
+        <CardContainer />
+      </div>
     </div>
```

Comment:
whatever your component is returning needs to either be wrapped in one element, or `be` one element, but you don't need the outer div here since it only has one child, the social-card div. something that wasn't discussed in TK but might come in handy is when you have to put multiple elements in a wrapper for your return statement, but the div doesn't need any classes or applied styles, you can use `React.Fragment` so there won't be an unnecessary div in your DOM tree